proxy/process.go

```
@@ -345,31 +345,31 @@ func (p *Process) stopCommand(sigtermTTL time.Duration) {
 	}
 
 	if err := p.terminateProcess(); err != nil {
-		p.proxyLogger.Infof("<%s> Failed to gracefully terminate process: %v", p.ID, err)
+		p.proxyLogger.Debugf("<%s> Process already terminated: %v (normal during shutdown)", p.ID, err)
 	}
 
 	select {
 	case <-sigtermTimeout.Done():
-		p.proxyLogger.Infof("<%s> Process timed out waiting to stop, sending KILL signal", p.ID)
+		p.proxyLogger.Debugf("<%s> Process timed out waiting to stop, sending KILL signal (normal during shutdown)", p.ID)
 		p.cmd.Process.Kill()
```

golangci-lint (1.64.8)

Error return value of `p.cmd.Process.Kill` is not checked

(errcheck)